apps/condo/domains/billing/schema/AllResidentBillingVirtualReceiptsService.test.js

```diff
@@ -227,7 +233,22 @@ describe('AllResidentBillingVirtualReceiptsService', () => {
             expect(receipts).toHaveLength(0)
         })
 
-        test('Do not return payments with processing statuses', async () => {
+        test('Return payments with init statuses', async () => {
```

Comment:
Best test names usually explain what happens in the test, maybe can filter against status=init ? or smth like that?